crates/uv/src/commands/project/run.rs

```diff
@@ -940,6 +942,11 @@ hint: If you are running a script with `{}` in the shebang, you may need to incl
         base_interpreter.sys_executable().display()
     );
 
+    warn_if_prerelease_python_upgrade_available(
```

Comment:
I think we'd want to put this somewhere in Python discovery instead? Like, when we choose to use an interpreter?